src/components/Text/Text.tsx

```diff
@@ -1,37 +1,80 @@
 import clsx from 'clsx';
-import React, { ReactNode } from 'react';
+import React from 'react';
 import styles from './Text.module.css';
 
-export interface Props {
-  /**
-   * The rendered tag name of the Heading. The tag name should always relate to the [document outline](http://html5doctor.com/outlines/) and a tag name should never be chosen for its default aesthetic qualities. If a specific style is desired, use the `size` prop to manipulate the Heading style.
-   */
-  as: 'p' | 'span';
-  /**
-   * The child node(s)
-   */
-  children?: ReactNode;
+export type Size =
+  | 'h1'
+  | 'h2'
+  | 'h3'
+  | 'h4'
+  | 'h5'
+  | 'body'
```

Comment:
No `md` size, but defaults to `body`, should `md` be added to follow recommendations?

Reply:
@jinlee93 we should talk more about this, but I think it's important to dedicate `Text` for non-heading level text (such as `p` and `span`). A `Heading` component should manage the heading-level semantics and styles. 

It would still be possible to achieve a `Text` component with a heading-type style using utility classes, but would recommend against building that into the component to not encourage that usage.

Reply:
Mmmmmmm I can't decide 😅 'body' is just such a common term when discussing text, but the text used for body is generally medium-sized, no? I think I like 'body' more because the usage is just much clearer. Kinda like 'overline' and 'caption', and even the headings – the size of the text isn't as important as how it's used.

Reply:
@bradfrost The `Text` component as-is can only render a `p` or a `span` semantically, but can be styled to look like anything (which is what this `Size` type controls).

I took a quick scan through the app, and people are styling Text to look like a heading pretty rarely, and if they do it's `"h3"` or `"h4"`. Are you suggesting instead of
```
<Text size="h3">
```
to encourage
```
<Text className="eds-u-typography-h3" >
```
instead?

Reply:
and I'd agree with keeping `body` 👍

Reply:
Kept body, but also added `lg` (available) and `md` (same as body)